x-pack/plugins/stack_connectors/public/connector_types/sentinelone/sentinelone_params.tsx

```diff
@@ -1,333 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React, { useCallback, useEffect, useMemo, useState, ReactNode } from 'react';
-import { reduce } from 'lodash';
-import {
-  EuiButtonIcon,
-  EuiComboBox,
-  EuiFieldText,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiFormRow,
-  EuiInMemoryTable,
-  EuiSuperSelect,
-} from '@elastic/eui';
-import {
-  ActionConnectorMode,
-  ActionParamsProps,
-  TextAreaWithMessageVariables,
-} from '@kbn/triggers-actions-ui-plugin/public';
-import { useSubAction, useKibana } from '@kbn/triggers-actions-ui-plugin/public';
-import { EuiBasicTableColumn, EuiSearchBarProps, EuiLink } from '@elastic/eui';
-import { SUB_ACTION } from '../../../common/sentinelone/constants';
-import type {
-  SentinelOneGetAgentsParams,
-  SentinelOneGetAgentsResponse,
-  SentinelOneGetRemoteScriptsParams,
-  SentinelOneGetRemoteScriptsResponse,
-  SentinelOneActionParams,
-} from '../../../common/sentinelone/types';
-import type { SentinelOneExecuteSubActionParams } from './types';
-import * as i18n from './translations';
-
-type ScriptOption = SentinelOneGetRemoteScriptsResponse['data'][0];
-
-const SentinelOneParamsFields: React.FunctionComponent<
```

Comment:
FYI: this component is not currently being used and unclear if it ever will, thus I have deleted it rather than attempt to fix it up given the changes in this PR. We can recover it in the future if needed